ding/utils/data/dataset.py

```diff
@@ -1,13 +1,21 @@
 from typing import List, Dict
 import pickle
 import torch
+import numpy as np
 from torch.utils.data import Dataset
 
+from ding.utils import DATASET_REGISTRY
 
+
+@DATASET_REGISTRY.register('naive')
```

Comment:
why not we merge `NaiveRLDataset` and `OfflineRLDataset` to one dataset class(`NaiveDataset`)

Reply:
ok